src/h5p-editor-csv-to-text.ts

```diff
@@ -145,6 +159,9 @@ class CSVToTextWidget extends H5PWidget<H5PFieldText> implements IH5PWidget {
 
     this.textarea.value = newValue;
     this.setValue(this.field, newValue);
+
+    const inputValueDiv = document.getElementById("csv-to-text-value") as HTMLDivElement;
+    inputValueDiv.innerHTML = files[0]?.name ?? "";
```

Comment:
If we want to, we could list out all the file names.

```suggestion
    inputValueDiv.innerHTML = files.map(({name}) => name).join(', ');
```

Reply:
Yes ✨ added now